requirements/edx/base.in

```diff
@@ -80,6 +80,7 @@ edx-django-sites-extensions
 edx-django-utils>=3.12.0            # Utilities for cache, monitoring, and plugins; 3.12.0+ for set_code_owner_attribute method
 edx-drf-extensions
 edx-enterprise
+edx-event-routing-backends
```

Comment:
Maybe in the next PR add a brief comment about the dependency? Something like `# Allows sending LMS events via xAPI and Caliper`